Formula/i/iipsrv.rb

```diff
@@ -0,0 +1,39 @@
+class Iipsrv < Formula
+  desc "High performance image server for high resolution and scientific images"
+  homepage "https://iipimage.sourceforge.io"
+  url "https://downloads.sourceforge.net/iipimage/IIP%20Server/iipsrv-1.2/iipsrv-1.2.tar.bz2"
+  sha256 "d2483313d62eb617d05f3a55c5f09551fe5165725faba63aca680cb87d56bf6c"
+  license "GPL-3.0-or-later"
+
+  depends_on "pkg-config" => :build
+  depends_on "fcgi"
+  depends_on "jpeg-turbo"
+  depends_on "libmemcached"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "openjpeg"
+  depends_on "webp"
+
+  def install
+    # Temporary fix for clang16
+    inreplace "configure", "TIFFGetVersion()", "int a"
+    system "./configure", *std_configure_args, "--disable-silent-rules"
+    system "make"
+    sbin.install "src/iipsrv.fcgi" => "iipsrv"
+    man8.install "man/iipsrv.8"
```

Comment:
Is there no `make install`?

Reply:
`make install` will install the man page, but not the executable. I'll submit a patch upstream for `make install` to behave as it should, but suggest we use explicit installs for Homebrew for both for this version

Reply:
The patches are kind of piling up here, which is against our policy: https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff

I think I'd rather wait for the next version then.

Reply:
> I think I'd rather wait for the next version then.

OK, no worries. I'll keep the formula to hand and re-submit a clean pull request when the next upstream release is made